packages/circuits/circuits/qr_verify.circom

```diff
@@ -5,11 +5,14 @@ include "./sha.circom";
 
 
 template QR_Verify(n, k, len) {
+    signal input signedMessage;
```

Comment:
Is signedMessage the correct name? 
How about calling it `signal`?

Reply:
I think this is some message prover want to approve. So I refer signedMessage.

Reply:
Yes, but its not "signed" by anyone technically right? Maybe we can simply call it `message`?

@Meyanis95 wdyt?

Reply:
Yes I'm advocating to just call it `signal`, as its role is to commit to a certain value (a signal) without actually "signing" it.

Reply:
signal also the key work of circom, how about signalMessage.

Reply:
We can use signalHash then
https://github.com/semaphore-protocol/semaphore/blob/0602554b97f156a1c203330af2af91b5fbd886e3/packages/circuits/semaphore.circom#L53